docs/components/networktransform.md

```diff
@@ -20,4 +16,30 @@ When enabled, extrapolation estimates the time between when a tick advances in s
 
 ## Interpolation
 
-When enabled, indicates the calculated interval between receiving and updating the position between two positions. It smoothes positions through calculations, for example EnableRange allows the position to update and return without causing snapping.
\ No newline at end of file
+Interpolation creates a smooth object transition by using two known, historical positions to predict an object's path during the interval between receiving and updating positional data for that object. This is a buffer for the network to sync between object input and rendering, and reduces object jitter or position snapping.
+
+You can select the best interpolation algorithm for the `GameObject` you want to sync. However, you still need to [interpolate your authoritative objects](../learn/clientside-interpolation.md).
+
+Our default interpolation algorithm is `BufferedLinearInterpolator`. This buffers values before making them available to the `NetworkTransform` value to update. `NetworkTransform` is able to accumulate jittered network values without affecting the transform state and then consume them at regular intervals.
+
+You can set global buffering by using:
+
+```csharp
+NetworkManager.Singleton.NetworkTimeSystem.ServerBufferSec = 0.2f;
+```
+
+:::note
+Use `BufferedLinearInterpolator` carefully to avoid excessive latency in your game! Although it will smooth transforms on jittery connections, it adds more latency to your transform updates.
+
+When targeting mobile platforms, you’ll want to use bigger buffer values while keeping interpolation times lower for platforms with more stable connections by keeping buffer time low.
+:::
+
+### Implementing Interpolation for NetworkTransform
+
+Use the `IInterpolator` interface to create your own custom interpolator and override the different interpolators in the `NetworkTransform` interpolator. The interpolator you implement must keep its own state and is given new values every time there are new values available to `NetworkTransform`'s `OnValueChanged`.
```

Comment:
current implementation removes this, let's come back to this doc once I merge my changes and they are commited to our main develop branch